pages/common/set.md

```diff
@@ -0,0 +1,19 @@
+# set
+
+> Display, set or unset value of shell attributes and positional parameters.
+
+- Display the names and values of shell variables:
+
+`set`
+
+- Mark variables that are modified or created for export:
+
+`set -a`
+
+- Notify of job termination immediately:
+
+`set -b`
+
+- Set various options, e.g. enable `vi` style line editing:
+
+`set -o vi`
```

Comment:
Perhaps `vi` should be surrounded by `{{` and `}}`?

Reply:
Ahhh... yes :(